src/js/view/azdata/azDataGraph.js

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I think it might be better to create a parallel folder structure under `azdata` directory so we don't need to move the existing content around.  For example, putting this in `azdata/view` and leaving the mxGraph `view` content where it is.  This will allow us to leave the existing content as-is while still separating out of new content.